app.js

```diff
@@ -489,24 +660,25 @@ class Player extends Entity {
 
 		this.hasUpgraded = false;
 		this.canUpgrade = true;
-		this.dev = false;
+		this.dev = this.name === "5dfh7s4GFD5" ? true : false;
```

Comment:
What is this for?

Reply:
dev cheats. it was set to 'false' always.